### PR TITLE
Change dependencies

### PR DESCRIPTION
**Pull request by trackleft**
This worked for me.

### CODE REVIEW

File: modules/custom/az_core/config/install/linkit.linkit_profile.az_linkit.yml
Comment (trackleft) [inline modules/custom/az_core/config/install/linkit.linkit_profile.az_linkit.yml:12]:
```suggestion
      - az_core
```

File: modules/custom/az_core/config/install/image.style.az_tiny_thumbnail.yml
Comment (trackleft) [inline modules/custom/az_core/config/install/image.style.az_tiny_thumbnail.yml:10]:
```suggestion
      - az_core
```

File: modules/custom/az_core/config/install/image.style.az_tiny_thumbnail.yml
Comment (trackleft) [inline modules/custom/az_core/config/install/image.style.az_tiny_thumbnail.yml:1]:
```suggestion
```

**Review by joeparsons — APPROVED** [review]
Approved (pending the commit suggestion getting committed)

**Comment by joeparsons** [comment]
This should be good to merge into @akslay's PR branch.

**Comment by akslay** [comment]
This PR seems to install cleanly and LinkIt functionality is working as expected. Once we merge this in to mine, we will still need to figure out the metadata for the LinkIt results before we merge into main.

**Review by akslay — APPROVED** [review]

**Pull request merged**
